hatch/src/cli/mod.rs

```diff
@@ -1,8 +1,5 @@
 pub mod commands;
 
-#[cfg(test)]
-mod tests;
-
 use clap::{App, AppSettings, Arg, ArgMatches};
```

Comment:
I believe that with the new cli.yml file, everything but the first line of this file can be removed!

Reply:
can you PR and remove it?

Reply:
Will do 👍